src/core/layertreemodel.h

```diff
@@ -39,7 +39,8 @@ class FlatLayerTreeModel : public QAbstractProxyModel
       Name,
       Visible,
       Trackable,
-      InTracking
+      InTracking,
+      IsGeometryLocked
```

Comment:
Can we rename this to "GeometryLocked" or "GeometryIsLocked"? (imperative form)
It currently is not in line with "InTracking" and others which also don't have an "Is" prefix.

Reply:
done!